app/src/main/res/layout/activity_dashboard_mobile.xml

```diff
@@ -32,8 +32,6 @@
                 android:layout_height="?attr/actionBarSize"
                 android:background="?colorPrimary"
                 android:clipToPadding="false"
-                android:elevation="8dp"
-                app:elevation="8dp"
```

Comment:
there is another xml for landscape with the same name, you need to remove it as well.

Reply:
You are right, I have just modified, tested and updated the PR